newIDE/app/src/SceneEditor/index.js

```diff
@@ -1709,6 +1710,36 @@ export default class SceneEditor extends React.Component<Props, State> {
       this.editorDisplay.forceUpdateInstancesPropertiesEditor();
   };
 
+  forceUpdateCustomObjectRenderedInstances = () => {
+    const { project, projectScopedContainersAccessor } = this.props;
+
+    const resourcesInUse = new gd.ResourcesInUseHelper(
+      project.getResourcesManager()
+    );
+    projectScopedContainersAccessor.forEachObject(object => {
+      if (project.hasEventsBasedObject(object.getType())) {
+        object.getConfiguration().exposeResources(resourcesInUse);
+      }
+    });
+    const objectResourceNames = resourcesInUse
+      .getAllImages()
```

Comment:
Surely all resources should be listed

Reply:
TODO: check if a custom object containing a child which is a 3D model is properly updated or not in the scene when the child 3D model is modified.

Reply:
I've check with a model file that was not yet loaded by the editor. I didn't notice any issue.
I guess we should leave it as it is until we find an issue.

Reply:
Ok for now. I think that if you replace the .glb file by another one or modify it in blender, this will still not be reloaded and something we need to fix for later.